server/bin/pbench-state-manager.py

```diff
@@ -51,7 +51,10 @@ def main(options):
         args = {}
         if options.create:
             args["owner"] = options.create
-        if options.controller:
+        if options.controller and options.create:
+            # The create operation requires both controller and name, whether
+            # explicitly or via path resolution. Attach, however, wants only
+            # path or name, so don't copy a controller parameter.
             args["controller"] = options.controller
```

Comment:
I suppose it would be more clear if the comment was *outside* the `if` ... since this is really explaining the `if` itself rather than the code inside the condition block. 😆